### PR TITLE
Fix inconsistency with recipientId - Closes #1639

### LINKED ISSUE #1639: Inconsistency with recipientId
### Expected behavior
- `recipientId` should always end with uppercase `L`
- schema validation should not allow transactions with `recipientId` that contains lowercase `l` (or convert them on the fly, maybe - but not recommended imo)
- all entries in `trs` table for column `recipientId` should be converted to uppercase `L` (we have about ~100 such transactions on mainnet) - those transactions will still be valid, as `l|L` is not a part of transaction signature (last character is skipped)
- SQL queries and indexes that have `UPPER("recipientId")` should be corrected to not contain `UPPER`

### Actual behavior
- transactions with lowercase `l` for `recipientId` are allowed and saved to `trs` table with lowercase `l`

### Steps to reproduce
N/A

### Which version(s) does this affect? (Environment, OS, etc...)
All

### PR DESCRIPTION
**Pull request by diego-G**
### Review checklist

* The PR solves #1639
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: schema/swagger.yml [hunk 1]
```diff
@@ -123,6 +123,7 @@ paths:
           description: Address of account
           type: string
           format: address
+          pattern: '^[0-9]+L$'
```
Comment (4miners) [inline schema/swagger.yml]: We shouldn't allow addresses that starts from `0`

Comment (vitaly-t) [reply schema/swagger.yml]: That would be: `^[1-9][0-9]*L$` :wink:

Comment (diego-G) [reply schema/swagger.yml]: Guys you're right but I've uses the same one we have in our z_schema: https://github.com/LiskHQ/lisk/blob/960d316c177f8c300d89c7df01f6ed2911ab9aba/helpers/z_schema.js#L88
We can tackle it in another issue to keep the scope of this PR. What do you think @karmacoma ?

Another point to highlight is the reason I have introduced this pattern attribute: `format: address` line presented several times in Swagger specs is useless. Correct me if I am wrong @nazarhussain

Comment (4miners) [reply schema/swagger.yml]: I think we should fix it as part of current issue.

Comment (karmacoma) [reply schema/swagger.yml]: I think out of scope of this particular issue, and that we should address separately.

**Comment by nazarhussain** [comment]
@diego-G @MaciejBaj @karmacoma We have to revert these changes. The `pattern` attribute is not needed in swagger file. As we are already handling the pattern with the help of `format` attribute, which is integrated with our `z_schema` helper and already matching the correct regular expression there. 

https://github.com/LiskHQ/lisk/blob/a4a7b298dcb98f932843748d8bbaed5dd87bd955/helpers/z_schema.js#L87-L89

**Pull request merged**